test/e2e/rc.go

```diff
@@ -26,17 +26,41 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/labels"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
-	"github.com/golang/glog"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
+var _ = Describe("ReplicationController", func() {
+	var (
```

Comment:
I know this is the syntax they use in their examples, but it seems silly to have a block for a single var. What about just writing var c *client.Client here?